consul/resource_consul_acl_master_token.go

```diff
@@ -71,6 +71,23 @@ func resourceConsulACLMasterTokenCreate(d *schema.ResourceData, meta interface{}
 }
 
 func resourceConsulACLMasterTokenRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*consulapi.Client)
+
+	id := d.Id()
+	log.Printf("[DEBUG] Reading ACL master token %q", id)
+
+	q := consulapi.QueryOptions{
+		Token: d.Get("token").(string),
+	}
+	_, _, err := client.ACL().TokenRead(id, &q)
+	if err != nil {
```

Comment:
Could you check for the error specifically like what is being done in https://github.com/terraform-providers/terraform-provider-consul/blob/d843086ae969d04e43dcdbe850ed1dadfe08a968/consul/resource_consul_prepared_query.go#L162-L169 ?

I fear that network issues, for example, would delete the resource unexpectedly.